src/AvaloniaEdit/CodeCompletion/CompletionAcceptAction.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace AvaloniaEdit.CodeCompletion;
+
+/// <summary>
+/// Defines the pointer action used to request the insertion of a completion item.
+/// </summary>
+public enum CompletionAcceptAction
```

Comment:
Feels like it could be a flags enum, with a possibility to set `None` option at least.

Reply:
The current options `PointerPressed`/`PointerReleased`/`DoubleTapped` are mutually exclusive and can't be combined as flags.

I am not sure if we want to allow `None`. That would prevent users from using the mouse for insertions. I suggest to only add `None` if there is practical use case.

The current defaults (`TextEditorOptions.CompletionAcceptAction == CompletionAcceptAction.PointerPressed`) ensure that the behavior in existing apps is unchanged.